client/extensions/woocommerce/index.js

```diff
@@ -36,123 +36,70 @@ const getStorePages = () => {
 			container: Dashboard,
 			configKey: 'woocommerce/extension-dashboard',
 			path: '/store/:site',
-			sidebarItem: {
-				icon: 'house',
-				isPrimary: true,
-				label: translate( 'Dashboard' ),
-				slug: 'dashboard',
-				showDuringSetup: true,
-			},
 		},
 		{
 			container: Products,
 			configKey: 'woocommerce/extension-products',
-			path: '/store/products/:site',
 			documentTitle: translate( 'Products' ),
-			sidebarItem: {
-				icon: 'product',
-				isPrimary: true,
-				label: translate( 'Products' ),
-				slug: 'products',
-				showDuringSetup: false,
-			},
+			path: '/store/products/:site',
 		},
 		{
 			container: ProductCreate,
 			configKey: 'woocommerce/extension-products',
-			path: '/store/product/:site',
-			parentPath: '/store/products/:site',
 			documentTitle: translate( 'New Product' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **61** times:
`translate( 'Product' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).